bootstrap.d/dev-lang.yml

```diff
@@ -30,6 +42,40 @@ tools:
     install:
       - args: ['make', 'install']
 
+  - name: host-rust
+    from_source: rust
+    sources_required:
+      - rust-libc
+    tools_required:
+      - host-llvm-toolchain
+    compile:
+      - args: |
+            cat << EOF > config.toml
+            changelog-seen = 2
+
+            [llvm]
+            targets = "X86"
+
+            [build]
+            target = ["x86_64-unknown-managarm-system"]
```

Comment:
If `x.py install` doesn't consider this at all, should we maybe just pass `--target` to build instead?

Reply:
It does — what was confusing me was that since `install` builds stage2, the stage1 compiler was being build as linux -> linux (as opposed to `build` which only builds stage1 and thus creates linux -> managarm). So I was stopping the build early, but it does in fact do the right thing when it starts building stage2.

Reply:
Alright.